### DISCUSSION TITLE
Can a nation being manipulated by another nation?

### DISCUSSION THREAD

**Discussion by ProblemsSender** [TownyAdvanced/Towny#5209] [Q&A] [answered]
Is there a colony system that built-in Towny where the player who owns a Nation(as a King) can own another player's Nation and make it a united nation?(and turning the nation(that was the colony) back to their original mayor state, not a king anymore)

If we do have it... wow that's promising! :P

**Comment by LlmDl** [comment, marked as answer]
Towny does not have any feature that lets a nation manipulate another nation. Towny does allow for 2 nations merging into 1 nation.

SiegeWar allows for a nation to attack, and if successful, occupy another town.

**Reply by ProblemsSender** [reply]
Wow that was fast, thanks for guiding me, sir.
